airbyte-integrations/connectors/source-plaid/source_plaid/source.py

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+import json
+import pathlib
+from abc import abstractmethod
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple, Union
+
+import plaid
+from airbyte_cdk.logger import AirbyteLogger
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from plaid.api import plaid_api
+from plaid.model.accounts_balance_get_request import AccountsBalanceGetRequest
+from plaid.model.transactions_get_request import TransactionsGetRequest
+
+from .plaid_requester import PlaidRequester
```

Comment:
@jrhizor I think this needs to be removed as well if the `PlaidRequester` is going to be removed